k8s/templates/sumo-secrets.yaml

```diff
@@ -1,54 +1,46 @@
-
 apiVersion: v1
 kind: Secret
 metadata:
```

Comment:
Should `dms-calculate-csat-metrics`, `dms-process-exit-surveys`, `es-http-auth`, and `es-urls` be removed from here too?

Since there's a bunch of other secrets missing from here (like the `es7_` settings) do we need this file at all?